net.filebot.Filebot.yml

```diff
@@ -0,0 +1,90 @@
+app-id: net.filebot.Filebot
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: filebot.sh
+separate-locales: false
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: zenity
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/zenity.git
+        tag: 3.41.0
+  - name: chromaprint
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: archive
+        url: https://github.com/acoustid/chromaprint/archive/v1.5.0.tar.gz
+        sha256: 5c8e0d579cb3478900699110aa961c1552a422a18741cf67dd62136b1b877c7b
+  - name: libzen
+    subdir: Project/GNU/Library
+    config-opts:
+      - --enable-shared
+      - --disable-static
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /lib/*.la
+    sources:
+      - type: archive
+        url: https://mediaarea.net/download/source/libzen/0.4.39/libzen_0.4.39.tar.xz
+        sha256: cded5d7774294e7d5b42768ceb285243709b6e99eec6e980a3cf7f0890dbb622
+  - name: libmediainfo
+    subdir: Project/GNU/Library
+    config-opts:
+      - --enable-shared
+      - --disable-static
+      - --with-libcurl
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /lib/*.la
+    sources:
+      - type: archive
+        url: https://mediaarea.net/download/source/libmediainfo/21.09/libmediainfo_21.09.tar.xz
+        sha256: 69be9444f6a03a1bdf56713ee31d56e987c502a7ec18eb54ee70fcc171b7f126
+  - name: filebot
+    ensure-writable:
+      - /bin
+    buildsystem: simple
+    build-commands:
+      - tar xf FileBot_4.9.5-aur.tar.xz
+      - cp -r usr/share/filebot/* ${FLATPAK_DEST}
+      - install -Dm755 filebot.sh ${FLATPAK_DEST}/bin/filebot.sh
+      - install -Dm644 net.filebot.Filebot.appdata.xml /app/share/appdata/net.filebot.Filebot.appdata.xml
+      - install -Dm644 net.filebot.Filebot.svg /app/share/icons/hicolor/scalable/apps/net.filebot.Filebot.svg
+      - install -Dm644 net.filebot.Filebot.desktop /app/share/applications/net.filebot.Filebot.desktop
+    sources:
+      - type: file
+        path: net.filebot.Filebot.appdata.xml
+      - type: file
+        path: filebot.sh
+      - type: file
+        path: net.filebot.Filebot.desktop
+      - type: file
+        path: net.filebot.Filebot.svg
+      - type: file
+        url: https://get.filebot.net/filebot/FileBot_4.9.5/FileBot_4.9.5-aur.tar.xz
```

Comment:
404 not found.

Reply:
Well, I build that version, seems like it was pulled for some reason https://get.filebot.net/filebot/
will bump to the next one...

Reply:
Yep, pulled due to regressions, not happy about that process
https://www.filebot.net/forums/viewtopic.php?p=57077#p57077